task02_iryna/src/view/SystemInitiator.java

```diff
@@ -0,0 +1,46 @@
+package view;
+
+import controller.TextReader;
+import dao.Paragraph;
+import dao.Sentense;
+import dao.Text;
+import service.TextService;
+
+import java.io.IOException;
+
+public class SystemInitiator {
+    public SystemInitiator() {
+        TextReader textReader = new TextReader();
+        TextService textService = new TextService();
+
+        try {
+            printResult(textReader, textService);
+        } catch (IOException e) {
+            System.out.println("There's something wrong");
+        }
+
+        printPar();
```

Comment:
Does the economy of 6 symbols is a good reason to name method `printPar()` instead of `printParagraph()`? )))